core/domain/services/graphql/data/mutations/DatetimeMutation.php

```diff
@@ -88,7 +88,11 @@ public static function prepareFields(array $input)
             $args['tickets'] = array_map('sanitize_text_field', (array) $input['tickets']);
         }
 
-        return $args;
+        return apply_filters(
+            'FHEE__EventEspresso_core_domain_services_graphql_data__datetime_mutation_args',
```

Comment:
filtern name doesn't quite match FQCN, so moving `_mutation` out of variable name and into the path

```suggestion
            'FHEE__EventEspresso_core_domain_services_graphql_data_mutations__datetime_args',
```